### PR TITLE
WoE: implement Shrewd Investor

### PR DESCRIPTION
**Pull request by strib**
Issue: #2902

### CODE REVIEW

File: server/game/cards/06-WoE/ShrewdInvestor.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const Card = require('../../Card.js');
+
+class ShrewInvestor extends Card {
```
Comment (romanogit) [inline server/game/cards/06-WoE/ShrewdInvestor.js:3]: Class name should be `ShrewdInvestor`

Comment (strib) [reply server/game/cards/06-WoE/ShrewdInvestor.js]: Oops, good catch.  My fingers have problems with that word for some reason.  Fixed!

File: server/game/cards/06-WoE/ShrewdInvestor.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    setupCardAbilities(ability) {
+        this.play({
+            optional: true,
+            gameAction: ability.actions.sequential([
+                ability.actions.gainAmber((context) => ({
+                    target: context.player.opponent
+                })),
+                ability.actions.capture({ amount: 4 })
+            ]),
+            message: '{0} uses {1} to cause {2} to gain 1 amber, and then capture 4 amber onto {0}',
```
Comment (romanogit) [inline server/game/cards/06-WoE/ShrewdInvestor.js:15]: I think you should remove this `onto {0}` as zero is the player's name

Comment (strib) [reply server/game/cards/06-WoE/ShrewdInvestor.js]: Yep, fixed.

**Comment by romanogit** [comment]
Is the DockerFile supported to be pushed?

**Comment by strib** [comment]
Sorry, no, I just noticed and reverted that.

That new `npm run build`` step really slows down my dev cycle though, I need to find a better way around it...

**Pull request merged**